src/test/java/core/basesyntax/service/strategy/ReturnHandlerTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReturnHandlerTest {
+    private Storage storage;
+    private BalanceHandler handler;
+    private ReturnHandler returnHandler;
+    private final String fruit = "apple";
+    private final int quantity = 20;
+    private final int addQuantity = 5;
```

Comment:
Constant naming

Reply:
I tried to do it with constants but my project wasn't building. it Said that something like wrong definition order.
But the order was correct.